theme/static/js/hs_file_browser.js

```diff
@@ -919,14 +919,28 @@ $(document).ready(function () {
         });
     });
 
-     // Download method
+    // Download method
     $("#btn-download, #fb-download").click(function () {
         var downloadList = $("#fb-files-container li.ui-selected");
+
+        // Remove previous temporary download frames
+        $(".temp-download-frame").remove();
+
         if (downloadList.length) {
-            for (var i = 0; i < downloadList.length; i++) {
-                var url = $(downloadList[i]).attr("data-url");
-                var fileName = $(downloadList[i]).children(".fb-file-name").text();
-                downloadURI(url, fileName);
+            // Workaround for Firefox and IE
+            if (jQuery.browser.mozilla == true || !!navigator.userAgent.match(/Trident\/7\./)) {
```

Comment:
Logically it makes sense, but it wouldn't be equivalent because null is returned by the match method. null should not be used in logic operations in javascript. For example, if you run the following line in your browser console (same code without !! operator)
`jQuery.browser.mozilla == true || navigator.userAgent.match(/Trident\/7\./)`
you'd get a null as result, rather than a Boolean.